sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Bindings/ServiceBusBinding.cs

```diff
@@ -53,7 +53,8 @@ public async Task<IValueProvider> BindAsync(BindingContext context)
             var entity = new ServiceBusEntity
             {
                 MessageSender = messageSender,
-                EntityType = _attribute.EntityType
+                ServiceBusEntityType = _attribute.ServiceBusEntityType,
+                JsonSerializerSettings = _messagingProvider.Options.JsonSerializerSettings
```

Comment:
Why do we need `JsonSerializerSettings` on the `ServiceBusEntity`?

Reply:
It's used in the UserTypeArgumentBindingProvider, but I will change to make it more consistent with other usages.